unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/NotificationBadge/Tests/UnreadNotificationBadgeShould.cs

```diff
@@ -110,4 +112,84 @@ public void NotCleanUnreadNotificationsBecauseOfDifferentUser()
             "Notificaton container should be activated");
         Assert.AreEqual("1", unreadNotificationBadge.notificationText.text, "Notification text should be 1");
     }
+
+    /************/
```

Comment:
not really a fan of this separators

Reply:
Ups! sorry, it was a mark that I put temporally only for me and I forgot to remove. Done!